### PR TITLE
N15: pGLOWrpg

### PR DESCRIPTION
**Pull request by roalyr**
_Part of #320 ("Newsletter 15")_

### CODE REVIEW

**Comment by roalyr** [comment]
#320

File: content/posts/newsletter-015/index.md [hunk 1]
```diff
@@ -60,6 +60,38 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+- Implementing the input autocomplete system.
+- Implementing save data import and parsing.
+- Implementing world navigation system.
+- Implementing rudimentary CLI graphics (for debugging and some future use).
+
+Main features of reported (pre-alpha) version are:
+
+- Ability to generate and explore one or many worlds (see previous news).
+
+For main feature reports and dev blogs follow [@pGLOWrpg twitter] on Twitter.
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-015/index.md:89]: `[@pGLOWrpg twitter] on Twitter` seems a bit redundant - might look nicer if this just said `[@pGLOWrpg] on Twitter`? Would also need to update the reference on line 92 if so.

**Review by 17cupsofcoffee — APPROVED** [review]
:+1:

**Review by ozkriff — APPROVED** [review]
:+1: (reduced the overview size as the project was already featured in previous newsletter issues, optimized the GIF)

**Comment by roalyr** [comment]
Thanks, I'll update my template for future use.

**Pull request merged**